index.js

```diff
@@ -21,7 +21,7 @@ var cache = [
 
 function leftPad (str, len, ch) {
   // convert `str` to `string`
-  str = str + '';
+  str = String(str);
```

Comment:
The tests would still pass without the change here

Reply:
The TypeError seems to be reproducible on the browser but not Node.js. Could you check this [JSFiddle](https://jsfiddle.net/xgj4srkv/)? Thank you.

Reply:
The error "TypeError: Cannot read property '2' of undefined" is not relevant. Its because `module` is not defined. I tried 

```js
NaN + ''
"NaN"
```

In my chrome console and it works.